.changeset/fluffy-hotels-yell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sveltejs/adapter-cloudflare-workers": patch
+"@sveltejs/adapter-cloudflare": patch
+---
+
+Update READMEs with additional **Comparison** notes
```

Comment:
don't need a changelog since there are no code changes. I sent a new PR without this file or the formatting changes